src/components/MultiSig/components/MultiSigLookup/MultiSigLookup.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState } from 'react';
+import { MultiSigLookupProps } from './types';
+import styles from '../../MultiSig.module.sass';
+import multiSigLookupStyles from './MultiiSigLookup.module.sass'
+import { StargateClient } from '@cosmjs/stargate';
+import { isValidSeiAddress } from '@sei-js/core';
+import { HiLightBulb } from '@react-icons/all-files/hi/HiLightBulb';
+import { useWallet } from '@sei-js/react';
+import { isMultisigThresholdPubkey, MultisigThresholdPubkey, pubkeyToAddress, createMultisigThresholdPubkey, Secp256k1Pubkey } from '@cosmjs/amino';
+import { toast } from 'react-toastify';
+import TableWithDelete from '../Utils/TableWithDelete';
+
+const MultiSigLookup = ({setMultiSigAccount, inputtedAccounts, setInputtedAccounts, multiSigThreshold, setMultiSigThreshold}: MultiSigLookupProps) => {
+    const [lookupMultiSig, setLookupMultiSig] = useState<boolean>(false);
+    const [createNewMultiSig, setCreateNewMultiSig] = useState<boolean>(false);
```

Comment:
instead of having multiple booleans here, maybe we can just have a string that determines the page that's currently open?

Reply:
Makes sense, just updated to select the component based on an enum